docs/explanation/package-model.rst

```diff
@@ -1,9 +1,196 @@
-.. _package-model:
-
-=============
 Package Model
 =============
 
-.. caution::
+Because :term:`Ubuntu` is based on the community-driven :term:`Debian` project,
+:term:`Ubuntu` uses the :term:`Debian` packaging model/format.
+
+There are :ref:`SourcePackages` and :ref:`BinaryPackages`.
+
+.. _SourcePackages:
+
+Source Packages
+---------------
+
+A *source package* contains the :term:`Source` material used to build one or
+more :ref:`BinaryPackages`.
+
+A *source package* is composed of
+
+- a :file:`.dsc` (*Debian Source Control*) file,
+- one or more compressed tar files, and 
+- optionally additional files depending on the type and format of the *source package*.
+
+The *Source Control* file contains metadata about the *source package*, for instance, 
+a list of additional files, name and version, list of the binary packages it produces, 
+dependencies, a digital signature and many more fields.
+
+.. note::
+  The article :doc:`/reference/debian-dir-overview` showcases the layout of 
+  an unpacked *source package*.
+
+Source Package Formats
+~~~~~~~~~~~~~~~~~~~~~~
+
+There exist multiple formats for how the :term:`Source` is packaged. The format of a 
+*source package* is declared in the :file:`debian/source/format` file. This file should
+always exist. If this file can not be found, the :ref:`format 1.0 <SourcePackageFormat_1.0>`
+is assumed for backwards compatibility, but :manpage:`lintian(1)` will warn you about it.
+
+.. tip::
+
+    If you don't know what source format to use, you should probably pick either 
+    :ref:`3.0 (quilt) <SourcePackageFormat_3.0quilt>` or
+    :ref:`3.0 (native) <SourcePackageFormat_3.0native>`.
+
+
+.. _NativeSourcePackages:
+
+(Non-)Native Source Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most cases, a software project is packaged by external contributors
+(:term:`Maintainers <Maintainer>`) only tangentially related to the software project. 
+Often, the *source package* has to do modifications to solve specific problems for its target
+:term:`Distribution`. The *source package* can, in these cases, be considered as its own
+software project, like a fork.
+
+Consequently, the :term:`Upstream` releases and *source package* releases do not always align.
+
+Native packages almost always originate from software projects designed with :term:`Debian`
+packaging in mind and have no independent existence outside its target :term:`Distribution`.
+Native packages do not differentiate between :term:`Upstream` releases and *source package* releases.
+Therefore, the version identifier of a native package does not have a 
+:term:`Ubuntu` or :term:`Debian` specific component.
```

Comment:
```suggestion
Native packages do not differentiate between Upstream releases and source package releases.
Therefore, the version identifier of a native package does not have an 
Ubuntu- or Debian-specific component.
```

Reply:
I'm not 100% sure I could define, after reading this, what a native package actually *is*.